### DISCUSSION TITLE
Did you see the Faker.js repo is empty?

### DISCUSSION THREAD

**Discussion by notfounnd** [mockoon/mockoon#622] [General]
I found out yesterday. I know that Mockoon has a fork, but @mockonn/commons-server seems to use the version available in npmjs.

https://github.com/marak/Faker.js/

https://www.npmjs.com/package/faker

https://github.com/mockoon/commons-server/blob/main/package.json#L43

Is there any risk of Faker.js disappearing from npmjs as well and affecting Mockoon?

![image](https://user-images.githubusercontent.com/57534340/148411707-6bc2977a-7894-411f-9651-5c6d02a648a3.png)

**Comment by 255kb** [comment]
Yes, this is bad news. 
We never updated to the latest version and we use 5.5.3 I think, which is pretty stable and still on NPM. 
I wish I had forked this version, but I guess it can still be found somewhere. 

Looking at NPM it seems it got reverted to 5.5.3 as the empty version 6.6.6 disappeared.
Normally, you can't unpublish older version that other projects depends on: https://docs.npmjs.com/policies/unpublish
🤞

**Comment by notfounnd** [comment]
If it helps, it's been a while since I left Faker.js aside and I've been using this one (also based on Faker.js) because it's a fork that has always received updates.

https://armindojr.github.io/pure-gen
https://github.com/armindojr/pure-gen

**Comment by 255kb** [comment]
Thanks for the info. 
I will see how things evolve. I would like to avoid breaking changes as much as possible 😊
